README.md

```diff
@@ -167,6 +167,15 @@ fastify.route({
 | `csrfOpts` |  The csrf options. See  [csrf](https://github.com/pillarjs/csrf).     |
 
 
+## Securing the secret
+
+The `secret` shown in above code is strictly just a sample. In all cases, you'd need to make sure that the `secret`
+- **Never** hard-coded in the code or `.env` files or anywhere in the repository
+- Stored in some external services like KMS, Vault or something similar
+- Read at run-time and supplied in this option
+- Is long enough
+- Is truly random sequence of characters
```

Comment:
be consistent with the wording. either begin the sentence in each point with "Is" or don't. I would not do it and end the introductory sentence with "is" instead:

> that the `secret` is:
> - ...
> - ...